assets/stylesheets/style.scss

```diff
@@ -1,6 +1,9 @@
 // External Dependencies
 @import '../../node_modules/react-virtualized/styles';
 
+$flag-icon-css-path: 'images/flags' !default;
+@import '../../node_modules/flag-icon-css/sass/flag-icon';
```

Comment:
I'm not crazy about this css dependency here. Have you thought of any alternatives?